packages/hdwallet-portis/src/portis.ts

```diff
@@ -0,0 +1,386 @@
+    return Promise.resolve()
+  }
+
+  public async hasOnDevicePinEntry(): Promise<boolean> {
+    return this.info.hasOnDevicePinEntry()
+  }
+
+  public async hasOnDevicePassphrase(): Promise<boolean> {
+    return this.info.hasOnDevicePassphrase()
+  }
+
+  public async hasOnDeviceDisplay(): Promise<boolean> {
+    return this.info.hasOnDeviceDisplay()
+  }
+
+  public async hasOnDeviceRecovery(): Promise<boolean> {
+    return this.info.hasOnDeviceRecovery()
+  }
+
+  public async hasNativeShapeShift(
+    srcCoin: Coin,
+    dstCoin: Coin
+  ): Promise<boolean> {
+    return false
+  }
+
+  public clearSession(): Promise<void> {
+    return this.portis.logout()
+  }
+
+  public ping(msg: Ping): Promise<Pong> {
+    // no ping function for Portis, so just returning Pong
+    return Promise.resolve({ msg: msg.msg })
+  }
+
+  public sendPin(pin: string): Promise<void> {
+    // no concept of pin in Portis
+    return Promise.resolve()
+  }
+
+  public sendPassphrase(passphrase: string): Promise<void> {
+    // cannot send passphrase to Portis. Could show the widget?
+    return Promise.resolve()
+  }
+
+  public sendCharacter(charater: string): Promise<void> {
+    // no concept of sendCharacter in Portis
+    return Promise.resolve()
+  }
+
+  public sendWord(word: string): Promise<void> {
+    // no concept of sendWord in Portis
+    return Promise.resolve()
+  }
+
+  public cancel(): Promise<void> {
+    // no concept of cancel in Portis
+    return Promise.resolve()
+  }
+
+  public wipe(): Promise<void> {
+    return Promise.resolve()
+  }
+
+  public reset(msg: ResetDevice): Promise<void> {
+    return Promise.resolve()
+  }
+
+  public recover(msg: RecoverDevice): Promise<void> {
+    // no concept of recover in Portis
+    return Promise.resolve()
+  }
+
+  public loadDevice (msg: LoadDevice): Promise<void> {
+    return this.portis.importWallet(msg.mnemonic)
+  }
+
+  public async ethSupportsNetwork (chainId: number = 1): Promise<boolean> {
+    return this.info.ethSupportsNetwork(chainId)
+  }
+
+  public async ethSupportsSecureTransfer (): Promise<boolean> {
+    return this.info.ethSupportsSecureTransfer()
+  }
+
+  public async ethSupportsNativeShapeShift (): Promise<boolean> {
+    return this.info.ethSupportsNativeShapeShift()
+  }
+
+  public async ethVerifyMessage (msg: ETHVerifyMessage): Promise<boolean> {
+    const signingAddress = await this.web3.eth.accounts.recover(msg.message, ('0x' + msg.signature), false)
+    return signingAddress === msg.address
+  }
+
+  public describePath (msg: DescribePath): PathDescription {
+    switch (msg.coin) {
+    case 'Ethereum':
+      return describeETHPath(msg.path)
+    default:
+      throw new Error("Unsupported path")
+    }
+  }
+
+  public ethNextAccountPath (msg: ETHAccountPath): ETHAccountPath | undefined {
+    // Portis only supports one account for eth
+    return undefined
+  }
+
+  public async isInitialized (): Promise<boolean> {
+    return false
```

Comment:
This is asking "does the wallet have a seed in it". Better to assume it does when we have no way of checking.
```suggestion
    return true
```